dbms/src/Flash/Coprocessor/AggregationInterpreterHelper.cpp

```diff
@@ -31,6 +30,20 @@ bool isFinalAggMode(const tipb::Expr & expr)
     return expr.aggfuncmode() == tipb::AggFunctionMode::FinalMode || expr.aggfuncmode() == tipb::AggFunctionMode::CompleteMode;
 }
 
+bool isSecondStageSum(const tipb::Expr & expr)
```

Comment:
`SecondStageSum` is not a very suitable name, because what we want here is to find out all the sum agg function who's input is `partial result`. Considering that an agg is converted to 3-stage agg by TiDB, then both the second and the third stage of sum need to return true here.

Reply:
> `SecondStageSum` is not a very suitable name, because what we want here is to find out all the sum agg function who's input is `partial result`. Considering that an agg is converted to 3-stage agg by TiDB, then both the second and the third stage of sum need to return true here.

I get it